bundler/spec/other/major_deprecation_spec.rb

```diff
@@ -587,8 +581,8 @@
       end
 
       it "prints a deprecation warning informing about its removal", :bundler => "< 3" do
-        expect(deprecations).to include("the `--outdated` flag to `bundle show` was undocumented and will be removed without replacement")
-      end
+       expect(deprecations).to include("the `--outdated` flag to `bundle show` was undocumented and will be removed without replacement")
+     end
```

Comment:
Actually, it's the indentation of the line that was causing the extra error. Reverting the message casing is fine though, since it's at least a different formatting issue than what we're fixing here. This is how this should look I believe.

```suggestion
        expect(deprecations).to include("the `--outdated` flag to `bundle show` was undocumented and will be removed without replacement")
      end
```

You can run `bin/rubocop` locally from the bundler folder to double check that the indentation is ok.

Reply:
done. thanks for the rubocop hint

Reply:
Thanks for fixing it!